### ISSUE TITLE
Script seems to not work[Tampermonkey]

### ISSUE THREAD

**Issue by AlcBrains** [zerodytrash/Simple-YouTube-Age-Restriction-Bypass#218]
### What happened?

Hey,
Test video https://www.youtube.com/watch?v=7YrmIIMupAM
The first time I load the video, it showed the player controls but it was stuck on loading. When I refresh the page, I get the usual "Verify Your Age" prompt. I'm running it through tampermonkey, 

### Platform

Userscript

### Version

2.5.9

### What browser are you on?

Firefox

### Browser version

117.0.1

### Link to the video

https://www.youtube.com/watch?v=7YrmIIMupAM

### About you

- [X] I am a developer and/or have knowledge in programming

**Comment by drunkwinter** [comment]
Cannot repro. Anything interesting in the console?

**Comment by AlcBrains** [comment]
I apologize for the size of  the file, I am not sure what I'm looking for so I pasted all warnings and errors on the console. 
[log.txt](https://github.com/zerodytrash/Simple-YouTube-Age-Restriction-Bypass/files/12711672/log.txt)

The logs you see are after I disabled other extensions (privacy badger, ublock origin) 

TL;DR version: 


 I see CSP related errors:
`Content-Security-Policy: The page’s settings blocked the loading of a resource at inline (“script-src”).`

A couple of POST requests to `https://play.google.com/log?format=json&hasfast=true&authuser=0`
Which return a 401 (I'm guessing these aren't relevant since they're on a log related endpoint)

A bunch of warnings regarding missing `SameSite` attribute value
A few of those: `The resource at “https://rr8---sn-vuxbavcx-5uid.googlevideo.com/generate_204” preloaded with link preload was not used within a few seconds. Make sure all attributes of the preload tag are set correctly.` 
which seem like the most relevant

and two of these :`MouseEvent.mozInputSource is deprecated. Use PointerEvent.pointerType instead.` 

Let me know if I can test more on my end

**Comment by drunkwinter** [comment]
Nothing in that log is related to the script. Also there is no indication it's running or else it would have logged something.

Try checking if the script is enabled, if it is, try toggling on/off Tampermonkey.
